docker-compose.yml

```diff
@@ -33,6 +36,7 @@ services:
     volumes:
       - ./static:/srv/static
       - ./site-static:/srv/site-static
+      - ./user-media/:/srv/user-media
```

Comment:
Please sync this branch with master, this change landed in master already, you might run into conflicts.